src/viur/core/request.py

```diff
@@ -444,7 +445,7 @@ def _route(self, path: str) -> None:
                 # someone tries to exploit unicode normalisation bugs)
                 raise errors.BadRequest()
 
-            if key.startswith("_"):  # Ignore keys starting with _ (like VI's _unused_time_stamp)
+            if not param_filter(key, value):
```

Comment:
```suggestion
            if param_filter and param_filter(key, value):
```
So we not call the lamda all the times